src/test/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProviderTest.java

```diff
@@ -188,36 +156,16 @@ public void urlsWithUserDefinedProvider() throws Exception {
         PreferredProviderUserProperty userProperty =
             new PreferredProviderUserProperty(TestUserDisplayURLProvider.class.getName());
         foo.addProperty(userProperty);
-        ACL.as(foo);
-
         MockFolder folder = rule.createFolder("my folder");
-        FreeStyleProject project = (FreeStyleProject) folder
-            .createProject(rule.jenkins.getDescriptorByType(FreeStyleProject.DescriptorImpl.class), "my job",
-                           false);
-        Run<?, ?> run = project.scheduleBuild2(0).get();
-
-        String root = DisplayURLProvider.get().getRoot();
-        assertEquals("http://localhost:" + rule.getLocalPort() + "/jenkins/", root);
-        assertEquals(root + "job/my%20folder/job/my%20job/1/" + TestUserDisplayURLProvider.EXTRA_CONTENT_IN_URL,
```

Comment:
Same as above, although in practice I think this case can only be hit if you use something like `authorize-project`.